projects/plugins/jetpack/_inc/client/components/apps-card/index.jsx

```diff
@@ -24,17 +23,40 @@ class AppsCard extends React.Component {
 		} );
 	};
 
-	dismissCard = () => {
-		this.props.dismissAppCard();
-		analytics.tracks.recordJetpackClick( {
-			target: 'apps-card',
-			button: 'dismiss',
-			page: this.props.location.pathname,
-		} );
-	};
+	getAppCards = () => (
+		<div className="jp-apps-card__apps-badges">
+			<AppsBadge
+				altText={ __( 'Google Play Store download badge.', 'jetpack' ) }
+				titleText={ __( 'Download the Jetpack Android mobile app.', 'jetpack' ) }
+				storeName="android"
+				storeLink="https://play.google.com/store/apps/details?id=com.jetpack.android&utm_source=jpdash&utm_medium=cta&utm_campaign=getappscard"
+				onBadgeClick={ this.trackDownloadClick }
+			/>
+			<AppsBadge
+				altText={ __( 'Apple App Store download badge.', 'jetpack' ) }
+				titleText={ __( 'Download the Jetpack iOS mobile app.', 'jetpack' ) }
+				storeName="ios"
+				storeLink="https://apps.apple.com/us/app/jetpack-wp-security-speed/id1565481562?pt=299112ct=jpdash&mt=8"
```

Comment:
While working on jetpack.com/mobile, it was requested we leverage this specific URL for SEO purposes. 
```suggestion
				storeLink="https://apps.apple.com/us/app/jetpack-website-builder/id1565481562?pt=299112ct=jpdash&mt=8"
```